docs/sources/dashboards/previews.md

```diff
@@ -0,0 +1,112 @@
+
+```
+[feature_toggles]
+# enable features, separated by spaces
+enable = dashboardPreviews
+```
+
+3. Save your changes. Grafana should reload automatically; we recommend restarting the Grafana server in case of any issues.
+
+Visit the dashboard search page to verify that the setup was successful. You should be able to see dashboard preview placeholders for all your existing dashboards after clicking on the _Show previews_ toggle at the top of page.
+
+{{< figure src="/static/img/docs/dashboards/previews-successful-setup.jpg"
+max-width="950px" animated-gif="/static/img/docs/dashboards/previews-successful-setup.gif" >}}
+
+Please check [Grafana server logs]({{< relref "../administration/configuration.md#log" >}}) in case you see any warnings after clicking on the toggle - errors logs produced by `dashboard_service` should provide more context.
+
+{{< figure src="/static/img/docs/dashboards/previews-unsuccessful-setup.png" max-width="950px" >}}
+
+## About the dashboard previews crawler
+
+The dashboard previews crawler is a background process that:
+
+1. [Prepares a list of dashboards to visit](#preparing-the-dashboard-list)
+2. [Visits and takes a screenshot of each dashboard](#rendering-previews)
+3. [Saves the screenshots in persistent storage](#saving-previews)
+
+The crawler can be configured via the main config file:
+
+```ini
+[dashboard_previews.crawler]
+# Number of dashboards rendered in parallel. Default is 6.
+thread_count =
+
+# Timeout passed down to the Image Renderer plugin. It is used in two separate places within a single rendering request:
+# First during the initial navigation to the dashboard and then when waiting for all the panels to load. Default is 20s.
+# This setting should be expressed as a duration. Examples: 10s (seconds), 1m (minutes).
+rendering_timeout =
+
+# Maximum duration of a single crawl. Default is 1h.
+# This setting should be expressed as a duration. Examples: 10s (seconds), 1m (minutes).
+max_crawl_duration =
+
+# Minimum interval between two subsequent scheduler runs. Default is 12h.
+# This setting should be expressed as a duration. Examples: 10s (seconds), 1m (minutes).
+scheduler_interval =
+```
+
+### Preparing the dashboard list
+
+During the initial crawler run, the list will contain all dashboards across all organizations.
+The list will have fewer elements during subsequent runs - it will contain only dashboards which have changed since taking the last preview and those which are brand-new.
+
+Modifying the dashboard is the only way of refreshing that dashboard's preview; previews do not have a set timeout after which they expire.
+
+### Rendering previews
+
+The crawler sends a render request to the Image Renderer for each dashboard in the list. The renderer is instructed to open the dashboard in _kiosk mode_ ([Grafana play example of kiosk mode](https://play.grafana.org/playlists/play/1?kiosk)), take a screenshot, and scale it down to a small, 320 x 240px thumbnail.
```

Comment:
```suggestion
The crawler sends a render request to the Image Renderer for each dashboard in the list. The renderer is then instructed to open the dashboard in _kiosk mode, take a screenshot, and scale it down to a small, 320 x 240px thumbnail. [Grafana play is an example of kiosk mode](https://play.grafana.org/playlists/play/1?kiosk)).
```

Reply:
That link points to an example of the kiosk mode in `play.grafana`, the `play.grafana` is not an example of a kiosk mode itself